packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -611,6 +629,51 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
     return textValue;
   };
 
+  /* @conditional-compile-remove(mention) */
+  // Adjust the selection range based on a mouse / touch interaction
+  const handleOnMove = useCallback(
+    (event) => {
+      // Should we do anything?
+      if (interactionStartPoint !== undefined) {
+        // And did selection change?
+        if (
```

Comment:
A little messy having the two nested `if` cause technically it looks like it can just be one giant `if`
```
if (interactionStartPoint &&
        // And did selection change?
          event.currentTarget.selectionStart !== null &&
          (event.currentTarget.selectionStart !== targetSelection?.start ||
            event.currentTarget.selectionEnd !== targetSelection?.end)
        )
```

Reply:
Indeed, updated.